.rubocop.yml

```diff
@@ -0,0 +1,27 @@
+AllCops:
+  Exclude:     
+    - 'db/**/*'     
+    - 'config/**/*'     
+    - 'script/**/*'
+    - 'test/**/*.rb'
+    - '**/*.rake'
+    - 'lib/tasks/auto_annotate_models.rake'
+    - 'bin/**/*'
+    - 'Rakefile'
+    - 'vendor/**/*'
+
+Style/Documentation:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Metrics/LineLength:
+  Max: 120
+
+ClassAndModuleChildren:
+  EnforcedStyle: nested
+
+Style/SymbolArray:
```

Comment:
I actually like the default setting of this one.

Reply:
according to the Ruby style guide: [https://shopify.github.io/ruby-style-guide/](url)
"Prefer the literal array syntax to %w..."

Reply:
and according to [our coding guidelines](https://github.com/WeConnect/style-guide/tree/master/ruby#percent-literals)....